pkg/users/account-create-dialog.js

```diff
@@ -299,7 +354,42 @@
                     caption: _("Create account with weak password"),
                     style: "warning",
                     clicked: () => {
-                        return passwd_check(true, state.real_name, state.user_name, state.password, state.password_confirm, state.locked);
+                        return passwd_check(
+                            true,
+                            false,
+                            state.real_name,
+                            state.user_name,
+                            state.password,
+                            state.password_confirm,
+                            state.locked,
+                            state.change_passw_force,
+                            state.uid,
+                            accounts,
+                            change
+                        );
+                    }
+                }
+            );
+        }
+        if (state.uid_exists) {
+            footer.actions.push(
+                {
+                    caption: _("Create account with non-unique UID"),
+                    style: "warning",
+                    clicked: () => {
+                        return passwd_check(
+                            false,
+                            true,
+                            state.real_name,
+                            state.user_name,
+                            state.password,
+                            state.password_confirm,
+                            state.locked,
+                            state.change_passw_force,
+                            state.uid,
+                            accounts,
+                            change
+                        );
```

Comment:
These 19 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18353-20230215-154634-5d29fa30-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)